src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -275,19 +203,6 @@ public synchronized void setMeta(ListeFilme listeFilme) {
         System.arraycopy(listeFilme.metaDaten, 0, metaDaten, 0, MAX_ELEM);
     }
 
-    public synchronized DatenFilm istInFilmListe(String sender, String thema, String titel) {
-        // prüfen ob es den Film schon gibt
-        // und sich evtl. nur die URL geändert hat
-        for (DatenFilm film : this) {
-            if (film.arr[DatenFilm.FILM_SENDER].equals(sender)
-                    && film.arr[DatenFilm.FILM_THEMA].equalsIgnoreCase(thema)
-                    && film.arr[DatenFilm.FILM_TITEL].equalsIgnoreCase(titel)) {
-                return film;
-            }
-        }
-        return null;
-    }
-
     public synchronized ListeFilme neueFilme(ListeFilme orgListe) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Rename method "neueFilme" to prevent any misunderstanding/clash with field "neueFilme" defined on line 57. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1845)